test/jdk/jdk/modules/etc/JdkQualifiedExportTest.java

```diff
@@ -71,6 +71,9 @@ static void check(ModuleDescriptor md) {
     static Set<String> KNOWN_EXCEPTIONS =
         Set.of("jdk.internal.vm.ci/jdk.vm.ci.services",
                "jdk.internal.vm.ci/jdk.vm.ci.runtime",
+               "jdk.internal.vm.ci/jdk.vm.ci.code",
+               "jdk.internal.vm.ci/jdk.vm.ci.hotspot",
+               "jdk.internal.vm.ci/jdk.vm.ci.meta",
                "jdk.jsobject/jdk.internal.netscape.javascript.spi");
```

Comment:
The order of packages in the original change is hotspot, meta, code. Maybe you should use exactly this? Other than that, looks reasonable.

Reply:
Sure, thanks for the catch!

Reply:
Fixed!